src/main/javascript/src/views/participation/distribution/ParticipationManualDistribution.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+    <h1 class="mb-5">
+      Select which students you would like for each condition.
+    </h1>
+
+    <!-- Conditions Section -->
+    <p>Conditions</p>
+    <v-expansion-panels class="v-expansion-panels--icon" flat>
+      <v-expansion-panel
+        v-for="(condition, index) in this.conditions"
+        :key="condition.conditionId"
+      >
+        <v-expansion-panel-header>
+          {{ condition.name }} ({{ arrayDataProxy[index].length }})
+        </v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <ListParticipants
+            :listOfParticipants="arrayDataProxy[index]"
+            :moveToOptions="getConditionNames"
+            :moveToHandler="moveToHandler"
+            :selectedOption="'' + index"
+          />
+        </v-expansion-panel-content>
+      </v-expansion-panel>
+    </v-expansion-panels>
+
+    <!-- Unassigned -->
+    <ListParticipants
+      :listOfParticipants="arrayDataProxy[this.getConditionNames.length - 1]"
+      :moveToOptions="getConditionNames"
+      :moveToHandler="moveToHandler"
+      :selectedOption="'' + (this.getConditionNames.length - 1)"
+    />
+
+    <v-btn
+      elevation="0"
+      class="mt-10"
+      color="primary"
+      @click="submitDistribution"
+      >Continue</v-btn
+    >
+  </div>
+</template>
+
+<script>
+import ListParticipants from "../../../components/ListParticipants.vue";
+import store from "@/store";
+import { mapGetters, mapActions } from "vuex";
+import { participantService } from "@/services";
```

Comment:
Let's continue to use single quotes (where possible) instead of double quotes in our JavaScript for consistency.

Reply:
Updating